src/stylesheets/elements/_typography.scss

```diff
@@ -33,6 +33,27 @@ a {
   }
 }
 
+// External link consider "effortless style approach":
```

Comment:
It's a little bit more complicated of an approach. 

1.) It would mess up button-styled anchor links without some kind of reset on them.
2.) We'd have to figure out the best way to manage the user's domain in `[href^="http:"]:not([href*="my-domain.com"])` which I was not really sure what approach we wanted (e.g. a Sass variable, etc..). I was thinking we could always enhance-up to this option. What do you think?